src/order/Order.java

```diff
@@ -0,0 +1,63 @@
+package order;
+
+import product.Pizza;
+import product.PizzaIngredients;
+
+import java.time.LocalTime;
+import java.util.Map;
+import java.util.Random;
+
+/**
+ * The implementation of Order class, which 
+
+ **/
+public class Order {
+  private final int orderNumber;
+  private final LocalTime orderTime;
+  private int customerNumber;
+  private Map<String, Pizza> pizzaItems;
+
+  Order(int customerNumber, Map<String, Pizza> pizzaItems) {
+    this.customerNumber = customerNumber;
+    this.orderNumber = generateOrderId();
+    this.pizzaItems = pizzaItems;
+    checkPizzaNames();
+    this.orderTime = LocalTime.now();
+  }
+
+  private int generateOrderId() {
+    return new Random().nextInt(99999);
+  }
+
+  private void checkPizzaNames() {
+    int index = 0;
+    for (String name : pizzaItems.keySet()) {
+      if (name.length() < 4 || name.length() > 20 || !name.matches("[a-zA-Z]+")) {
```

Comment:
in real-world tasks, we call numbers in code like 4, 20, 99999 magic numbers, besides code author nobody knows what they mean, so it is better to have constants in the class and name them.
e.g. private static final int PIZZA_NAME_MIN_LENGTH = 4; etc.